### ISSUE TITLE
Term-Readline: status in core-p7 branch

### ISSUE THREAD

**Issue by jkeenan** [atoomic/perl#26]
In this ticket we will track our core-p7 progress on `dist/Term-Readline`.

As of the morning of Jul 08 2020, we are getting this output from `make test` for this distro.

```
t/../dist/Term-ReadLine/t/AE ....................................... skipped
t/../dist/Term-ReadLine/t/AETk ..................................... skipped
t/../dist/Term-ReadLine/t/ReadLine-STDERR .......................... Bareword found where operator expected at ../../lib/Term/ReadLine.pm line 379, near "Tgetent Term::Cap"
	(Do you need to predeclare Tgetent?)
#   Failed test 'use Term::ReadLine;'
#   at t/ReadLine-STDERR.t line 20.
#     Tried to use 'Term::ReadLine'.
#     Error:  syntax error at ../../lib/Term/ReadLine.pm line 379, near "Tgetent Term::Cap "
# syntax error at ../../lib/Term/ReadLine.pm line 397, near "}"
# Bareword "LoadTermCap" not allowed while "strict subs" in use at ../../lib/Term/ReadLine.pm line 389.
# Compilation failed in require at t/ReadLine-STDERR.t line 20.
# BEGIN failed--compilation aborted at t/ReadLine-STDERR.t line 20.
Can't locate object method "devtty" via package "Term::ReadLine" at t/ReadLine-STDERR.t line 22.
# Looks like your test exited with 2 just after 2.
FAILED at test 1
t/../dist/Term-ReadLine/t/ReadLine ................................. Bareword found where operator expected at ../../lib/Term/ReadLine.pm line 379, near "Tgetent Term::Cap"
	(Do you need to predeclare Tgetent?)
#   Failed test 'use Term::ReadLine;'
#   at t/ReadLine.t line 16.
#     Tried to use 'Term::ReadLine'.
#     Error:  syntax error at ../../lib/Term/ReadLine.pm line 379, near "Tgetent Term::Cap "
# syntax error at ../../lib/Term/ReadLine.pm line 397, near "}"
# Bareword "LoadTermCap" not allowed while "strict subs" in use at ../../lib/Term/ReadLine.pm line 389.
# Compilation failed in require at t/ReadLine.t line 16.
# BEGIN failed--compilation aborted at t/ReadLine.t line 16.
Bareword found where operator expected at t/ReadLine.t line 19, near "new Term::ReadLine"
	(Do you need to predeclare new?)
String found where operator expected at t/ReadLine.t line 19, near "Term::ReadLine 'test term::readline'"
	(Do you need to predeclare Term::ReadLine?)
syntax error at t/ReadLine.t line 19, near "new Term::ReadLine "
Global symbol "$t" requires explicit package name (did you forget to declare "my $t"?) at t/ReadLine.t line 21.
Execution of t/ReadLine.t aborted due to compilation errors.
# Looks like your test exited with 255 just after 1.
FAILED at test 1
t/../dist/Term-ReadLine/t/Tk ....................................... skipped


```
These results were generated on an unthreaded build on Linux.  The results might differ significantly if run on a threaded build.  There have been subsequent commits to this branch which may have impacted these results.  YMMV.

**Comment by atoomic** [comment]
I think this was fixed by 2785301ad167ec22e0c0eeb5bf9b13c86022a434
need to confirm then we can close this case

**Comment by jkeenan** [comment]
> I think this was fixed by [2785301](https://github.com/atoomic/perl/commit/2785301ad167ec22e0c0eeb5bf9b13c86022a434)
> need to confirm then we can close this case

I suspected that as well, as these results were generated much earlier today.  Will check tomorrow.

**Comment by jkeenan** [comment]
> I think this was fixed by [2785301](https://github.com/atoomic/perl/commit/2785301ad167ec22e0c0eeb5bf9b13c86022a434)
> need to confirm then we can close this case

Yes, as of v5.32.0-576-g9e5ecda8ae, Term-Readline is testing clean.  Thanks for your investigation.
